Neo4j.Driver/Neo4j.Driver/Internal/Routing/RoutingTableManager.cs

```diff
@@ -329,7 +329,7 @@ public async Task<IRoutingTable> UpdateRoutingTableAsync(ISet<Uri> triedUris = n
                     catch (Exception e)
                     {
                         _logger?.Info(
```

Comment:
This feels more like `Warn` than `Info` and can receive exception, not just the message